### DISCUSSION TITLE
Feature Not Found.

### DISCUSSION THREAD

**Discussion by fabiopesaju** [freeplane/freeplane#1461] [General]
Excuse my ignorance, but does this feature of listing all nodes in a tree form, as in VYM, also exist somewhere in FreePlane?

![image](https://github.com/freeplane/freeplane/assets/20627847/bea711c9-26cd-4767-8575-9462b8270dcb)

**Comment by sifran-github** [comment]
you can do this with a combination of:

- view -> new map view
- view -> view settings -> outline view

**Reply by fabiopesaju** [reply]
thank you. I use vym, I'm trying to get to know freeplane better and redo my maps with it
